docs/source/contributing/install-docker.md

```diff
@@ -1,2 +1,8 @@
 Install [Docker Desktop](https://docs.docker.com/get-docker/) for your operating system.
 Docker Desktop includes all Docker tools.
+
+It's required that the user you will run install script with be in the docker group.
+You can add the user to the group by running (from the user shell) :
+```
+usermod -a -G docker $USER && newgrp docker
+```
```

Comment:
This solution is only relevant on some operating systems. I think it would make more sense to link to Docker's own documentation of this topic: https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user